src/tools/annotation/FreehandMouseTool.js

```diff
@@ -1032,11 +1029,8 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
   _startDrawing(evt) {
     const eventData = evt.detail;
     const measurementData = this.createNewMeasurement(eventData);
-    const element = eventData.element;
+    const { element } = eventData;
     const config = this.configuration;
-
-    this._referencedElement = element;
```

Comment:
This wasn't used anywhere. It seems like it would be expedite grabbing the element throughout but I'm not sure we want to save this type of information from event to event.

There's also this._activeDrawingToolReference, which caches a handle for the data, which looks like it was intended to do the same thing but isn't used consistently. Probably best to replace with a helper function that takes the element or event as an argument.

Reply:
Huh, interesting, `newImageCallback` uses the `_activeDrawingToolReference`, `_referencedElement` can indeed be nuked.

> Probably best to replace with a helper function that takes the element or event as an argument.

The difficulty here is that the event will contain the new imageId, etc. What's needed is a reference to the actively drawn data, so that it can be closed cleanly if you are part way through the drawing loop and decide to change the image, either by scrolling or dragging a new scan to the canvas.

Reply:
Sounds like I'll have to investigate more with `this._activedrawingToolReference` if we're going to do anything with that (it would definitely be a separate PR). For this, I was trying to keep it pretty small just to keep continuity and make the review pretty straightforward.